### PR TITLE
Made  package for it

### PR DESCRIPTION
**Pull request by AjayAntoIsDev**
just run pip install . to install the package
(Make a issue if anything goes wrong I will try to help)

### CODE REVIEW

**Comment by eggnaut** [comment]
Hi! Thanks for making the package files. Unfortunately, I have no idea about creating Python packages, so I'll have to educate myself on them (Youtube... LOL). I've seen your fork and it looks good, however for me to merge the branches, I need to understand the files. I'll get back to you when I'm ready to create a Python package, thank you!

File: EggEngine.egg-info/PKG-INFO [hunk 1]
```diff
@@ -0,0 +1,9 @@
+Metadata-Version: 2.1
+Name: EggEngine
+Version: 0.0.1
+Summary: A custom framework/library written with and for Python.
+Home-page: https://github.com/eggnaut/EggEngine
+Download-URL: https://github.com/eggnaut/EggEngine
+Author: EggNaut
```
Comment (eggnaut) [inline EggEngine.egg-info/PKG-INFO:7]: Author: eggnaut

File: EggEngine.egg-info/requires.txt [hunk 2]
```diff
@@ -0,0 +1,3 @@
+cryptography
+Pillow
+setuptools
```
Comment (eggnaut) [inline EggEngine.egg-info/requires.txt:3]: pygame is also a module that EggEngine uses for EggEngine.image

**Review by eggnaut — CHANGES_REQUESTED** [review]

File: EggEngine.egg-info/PKG-INFO (hunk 1 above)
Comment (AjayAntoIsDev) [reply EggEngine.egg-info/PKG-INFO:7]: 👍

File: EggEngine.egg-info/requires.txt (hunk 2 above)
Comment (AjayAntoIsDev) [reply EggEngine.egg-info/requires.txt:3]: You have to change it in setup.py

Comment (eggnaut) [reply EggEngine.egg-info/requires.txt:3]: okay got it, thanks!

**Review by eggnaut — APPROVED** [review]

**Comment by AjayAntoIsDev** [comment]
Oops i accidentally deleted the changes

**Comment by eggnaut** [comment]
Oh thats fine lol, you might be able to recover them I'm not sure.

**Comment by AjayAntoIsDev** [comment]
Yea i have the files in my local machine

**Comment by eggnaut** [comment]
Oh okay thats good

**Comment by AjayAntoIsDev** [comment]
I will make a new pull request

**Comment by eggnaut** [comment]
I re-opened this PR, you can continue here.

**Review by eggnaut — APPROVED** [review]

**Comment by AjayAntoIsDev** [comment]
You can merge these changes if you want to

**Comment by eggnaut** [comment]
> You can merge these changes if you want to

Will do when I'm ready, thank you for all your help!

**Comment by AjayAntoIsDev** [comment]
👌

**Pull request closed without merging**